scripts/archlinux/solidity/PKGBUILD

```diff
@@ -0,0 +1,63 @@
+# Maintainer: Afri 5chdn <aur@cach.co>
+# Available from AUR: https://aur.archlinux.org/packages/solidity/
+
+pkgname=solidity
+pkgver=0.4.2
+pkgrel=1
+_chash='4fc6fc2ca59579fae2472df319c2d8d31fe5bde5'
+pkgdesc="The Solidity Contract-Oriented Programming Language (Including solc and lllc)"
+arch=('i686' 'x86_64')
+depends=(
+  'boost'
+  'boost-libs'
+  'crypto++'
```

Comment:
Same comment as above. Actually tested it and the package is built without this dependency.